docker-compose.yml

```diff
@@ -13,12 +13,9 @@ services:
     ports:
       - "5000:5000"
     volumes:
-      - node_modules:/app/node_modules
       - .:/app:delegated
+      - /app/node_modules
```

Comment:
@luqmaan The way I understood it via @ifsimicoded is that the way we have it without the `:` followed a reference directory effective mounts it nowhere. Could be way off on this though.

Reply:
I was referencing the section [The node_modules Volume Trick](http://jdlm.info/articles/2016/03/06/lessons-building-node-app-docker.html). 
I think what's happening is node_modules is mounted from the image, and not the host, so they're not blown away when the image is run.

Reply:
But honestly, docker-compose syntax isn't familiar to me, so there's probably a better way that persists these things across docker-compose ups and downs.